compatibility/krastorio2/data-updates.lua

```diff
@@ -31,78 +28,114 @@ util.add_research_ingredient("fluid-handling", "logistic-science-pack")
-  else
-    ff_restack(item_name)
+-- Remove Transport Science pack from Singularity lab
+for i, input in pairs(data.raw.lab["kr-singularity-lab"].inputs) do
+  if input == "ff-transport-science-pack" then
+    table.remove(data.raw.lab["kr-singularity-lab"].inputs, i)
   end
+end
 
-  deadlock_crating.add_crate_autotech(item_name, item.tech)
-end]]
+-- Custom stcks size (will be halved in final-fixes)
+data.raw.item["wood"].stack_size = 50
+data.raw.item["sand"].stack_size = 50
+data.raw.item["raw-imersite"].stack_size = 30
+data.raw.item["raw-rare-metals"].stack_size = 30
+data.raw.item["coke"].stack_size = 50
+data.raw.item["glass"].stack_size = 100
+data.raw.item["fertilizer"].stack_size = 100
+data.raw.item["biomass"].stack_size = 100
+data.raw.item["quartz"].stack_size = 100
+data.raw.item["silicon"].stack_size = 100
+data.raw.item["rare-metals"].stack_size = 100
+data.raw.item["imersium-plate"].stack_size = 100
+data.raw.item["enriched-iron"].stack_size = 100
+data.raw.item["enriched-copper"].stack_size = 100
+data.raw.item["enriched-rare-metals"].stack_size = 100
+data.raw.item["lithium-chloride"].stack_size = 100
+data.raw.item["lithium"].stack_size = 100
+data.raw.item["sulfur"].stack_size = 100
+data.raw.item["lithium-sulfur-battery"].stack_size = 100
+data.raw.item["explosives"].stack_size = 100
+data.raw.item["imersite-powder"].stack_size = 100
+data.raw.item["tritium"].stack_size = 100
+data.raw.item["fuel"].stack_size = 50
+data.raw.item["bio-fuel"].stack_size = 50
+data.raw.item["advanced-fuel"].stack_size = 50
+data.raw.item["iron-stick"].stack_size = 200
+data.raw.item["iron-gear-wheel"].stack_size = 200
+data.raw.item["rocket-control-unit"].stack_size = 50
+data.raw.item["low-density-structure"].stack_size = 50
+data.raw.item["rocket-fuel"].stack_size = 50
+data.raw.item["matter-stabilizer"].stack_size = 50
+data.raw.item["charged-matter-stabilizer"].stack_size = 50
+data.raw.item["processing-unit"].stack_size = 200
+data.raw.item["engine-unit"].stack_size = 200
+data.raw.item["electric-engine-unit"].stack_size = 200
+data.raw.item["flying-robot-frame"].stack_size = 200
+data.raw.item["imersite-crystal"].stack_size = 200
+data.raw.item["uranium-235"].stack_size = 200
+data.raw.item["uranium-238"].stack_size = 200
+data.raw.item["energy-control-unit"].stack_size = 200
+data.raw.item["space-research-data"].stack_size = 200
+data.raw.item["kr-black-reinforced-plate"].stack_size = 100
+data.raw.item["kr-white-reinforced-plate"].stack_size = 100
+data.raw.item["kr-steel-pipe"].stack_size = 100
+data.raw.ammo["explosive-turret-rocket"].stack_size = 50
+data.raw.ammo["nuclear-turret-rocket"].stack_size = 50
+data.raw.ammo["antimatter-turret-rocket"].stack_size = 50
+data.raw.ammo["artillery-shell"].stack_size = 50
+data.raw.ammo["nuclear-artillery-shell"].stack_size = 50
+data.raw.ammo["antimatter-artillery-shell"].stack_size = 50
+data.raw.ammo["atomic-bomb"].stack_size = 50
+data.raw.ammo["antimatter-rocket"].stack_size = 50
```

Comment:
I don't at all mind this approach, I'm just checking that you're not reinventing the wheel of the K2 stack size setting that we forcibly turn off? https://github.com/tburrows13/FreightForwarding/blob/master/compatibility/krastorio2/settings.lua

If you're deliberately doing it custom so that you've got perfect granularity, then no problem :)